ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-4.14.yaml

```diff
@@ -78,6 +78,28 @@ tests:
         MIRROR_IMAGES=true
     workflow: agent-e2e-generic
   timeout: 6h0m0s
+- as: e2e-metal-assisted-static-ip-suite
+  optional: true
+  run_if_changed: ^(data/.*)$
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        STATIC_IPS=true
+        TEST_FUNC=test_networking
+      PACKET_PLAN: m3.large.x86
+    workflow: assisted-baremetal
+- as: e2e-metal-assisted-bond
+  optional: true
+  run_if_changed: ^(data/.*)$
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        IS_BONDED=true
+        TEST_FUNC=test_networking
+      PACKET_PLAN: m3.large.x86
+    workflow: assisted-baremetal
```

Comment:
we don't use directly these ocp branches. The job should be defined in the `__edge` file, in the same way we define `e2e-metal-assisted-4-14` for example.